src/SprykerEco/Zed/AfterPay/Communication/Plugin/Oms/Command/CapturePlugin.php

```diff
@@ -23,6 +23,8 @@
 class CapturePlugin extends AbstractPlugin implements CommandByOrderInterface
 {
     /**
+     * {@inheritDoc}
+     *
```

Comment:
* - Sends payment capture request to AfterPay gateway, to capture payment for a specific order item.
* - If it is the first item capture request for given order, captures also full expense amount.
* - Saves the transaction result in DB and updates payment with new total captured amount.